app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -147,7 +148,7 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), UserInteractionHan
         if (currentGuid == BookmarkRoot.Root.id &&
             requireComponents.backgroundServices.accountManager.authenticatedAccount() == null
         ) {
-            view?.let { SignInView(it.bookmarkLayout, findNavController()) }
+            signInView = view?.let { SignInView(it.bookmarkLayout, findNavController()) }
```

Comment:
minor nit, but instead of adding/removing, can we just flip the visibility of it? `View.GONE` & `View.VISIBLE`?